packages/expo/package.json

```diff
@@ -94,6 +94,7 @@
     "expo-sensors-interface": "~1.1.0",
     "expo-sms": "~1.1.0",
     "fbemitter": "^2.1.1",
+    "fbjs": "^0.8.17",
```

Comment:
Let's work on removing this; it should be easy to do. RN is also working on removing fbjs.